Index: app/build/kotlin/compileDebugKotlin/caches-jvm/lookups/counters.tab
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>10\r\n0
===================================================================
diff --git a/app/build/kotlin/compileDebugKotlin/caches-jvm/lookups/counters.tab b/app/build/kotlin/compileDebugKotlin/caches-jvm/lookups/counters.tab
--- a/app/build/kotlin/compileDebugKotlin/caches-jvm/lookups/counters.tab	
+++ b/app/build/kotlin/compileDebugKotlin/caches-jvm/lookups/counters.tab	
@@ -1,2 +1,2 @@
-10
+11
 0
\ No newline at end of file
